extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -52,49 +35,38 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 		return track;
 	}
 
-	function onKeyUp (event) {
-		if (event.keyCode === 27) {
-			close();
-		}
-	}
-
 	function close (event) {
 		if (event) {
 			event.preventDefault();
 		}
-
-		if (modal) {
-			track({
-				action: Wikia.Tracker.ACTIONS.CLOSE,
-				label: 'username-login-modal'
-			});
-			$blackout.remove();
-			isOpen = false;
-		}
-
-		$(window).off('.authModal');
-	}
-
-	function onPageLoaded () {
-		if (modal) {
-			$(modal).removeClass('loading');
+		if (authPopUp) {
+			authPopUp.close();
 		}
+		$(window).off('.authPopUp');
 	}
 
-	function loadPage (url, onPageLoaded) {
-		var authIframe = window.document.createElement('iframe'),
-			modalParam = 'modal=1';
+	function loadPage (url) {
+		var modalParam = 'modal=1',
+			pageWidth = window.innerWidth,
+			popUpHeight = 670,
+			popUpMaxWidth = 768,
+			popUpWidth = pageWidth < popUpMaxWidth ? pageWidth : popUpMaxWidth,
+			popUpLeft = window.screenX + (pageWidth / 2) - (popUpWidth / 2),
+			popUpTop = window.screenY + (window.innerHeight / 2) - (popUpHeight / 2),
+			src = url + (url.indexOf('?') === -1 ? '?' : '&') + modalParam;
 
-		authIframe.src = url + (url.indexOf('?') === -1 ? '?' : '&')  + modalParam;
-		//for the selenium tests:
-		authIframe.id = 'auth-modal-iframe';
-		authIframe.onload = function () {
-			if (typeof onPageLoaded === 'function') {
-				onPageLoaded();
-			}
-		};
-		modal.appendChild(authIframe);
+		authPopUp = window.open(src, '_blank', 'width=' + popUpWidth + ',height=' + popUpHeight + ',top=' + popUpTop + ',left=' + popUpLeft);
```

Comment:
I would consider creating a separate method like getPopupSpecs() so we can minimize a size of this method and make it clearer what's happening here. Also, modalParam, popUpHeight and popUpMaxWidth look like consts so they could be moved to the class level.